src/Features/Core/Portable/EditAndContinue/ManagedEditAndContinueCapability.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.EditAndContinue
+{
+    /// <summary>
+    /// The capabilities that the runtime has with respect to edit and continue
+    /// </summary>
+    internal enum ManagedEditAndContinueCapability
```

Comment:
>ManagedEditAndContinueCapability [](start = 18, length = 32)

This could just be a flags enum. If we run out of bits we can make it bit array, but for now it'd be simpler.